sound/soc/sof/ipc3-priv.h

```diff
@@ -28,6 +28,8 @@ int sof_ipc3_validate_fw_version(struct snd_sof_dev *sdev);
 /* dtrace position update */
 int ipc3_dtrace_posn_update(struct snd_sof_dev *sdev,
 			    struct sof_ipc_dma_trace_posn *posn);
+/* RX handler for fuzzing purposes only */
+void sof_ipc3_do_rx_work(struct snd_sof_dev *sdev, struct sof_ipc_cmd_hdr *hdr, void *msg_buf);
```

Comment:
This comment is not quite accurate, the `sof_ipc3_do_rx_work()` is used by `sof_ipc3_rx_msg()` at this point only and it is the actual rx handler.